### DISCUSSION TITLE
Thank you! 🙌🏻 Can I help?

### DISCUSSION THREAD

**Discussion by lax4mike** [ramda/types#20] [General]
I was reading through [the proposal](https://github.com/ramda/ramda/discussions/3329) that @Harris-Miller wrote up and I'm in a similar boat. I started at a new company recently, and I'm trying to convince the team to embrace ramda.  We also just adopted typescript, which is making my case for ramda very difficult because [even doing very simple things is super painful](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAKjgQwM5wEpwGZQiOAIimRABNlCBuAKBoGMIA7VebYAGwFM4BeOAN5wmpLgC4iACy4cOEAHRgmAc0IBfWg2ascnLgFEAHjC4tgzPpnmNwEVFwAUNOFbJdsyAK4cYAFQgOhL6SwOjs3IQAlAA0cAD0cXBiTi4Y8kwwkg4AjDHOViDIMPRZcfIIADryDuWRcXmpinhgAPJQgYSxhCIgXFE0kQ7hXJFAA) and honestly could be a deal breaker for us using ramda in the new code that we're writing in typescript.

It's great that this repo is taking the initiative to remedy this glaring issue!

I'm an experienced developer, with about a year or more of typescript, so let me know if I can help in any way!

**Comment by Harris-Miller** [comment]
Yes! Help would be very much appreciated. Please check out The Issue's page and current MRs for starters. I haven't done it yet but I am planning on writing up a guide on how to write types and what minimum testing requirements should be. For example, there area lot of typing that currently work stand-alone, but don't when used in conjunction with `compose` like in your example.

Feel free to start with any functions that we don't already have an MR out for that you'd like to tackle. Your PRs should target the `develop` branch
